test/test-cdrafter.cc

```diff
@@ -0,0 +1,42 @@
+#include "test-drafter.h"
+
+#include "cdrafter.h"
+
+#include <string.h>
+
+
+TEST_CASE("c-interface parses blueprint correctly","[c-interface]")
+{
+    it_fixture_files fixture = it_fixture_files("test/fixtures/annotations-with-warning");
```

Comment:
Won't this test fail on windows since their file system path notation is different? So, reading from "test/fixtures" will give an error.